module/VuFind/src/VuFind/Net/UserIpReader.php

```diff
@@ -48,18 +48,19 @@ class UserIpReader
     protected $server;
 
     /**
-     * Should we respect the X-Forwarded-For header?
+     * Configuration specifying allowed HTTP headers containing IPs (false for none).
+     * See [Proxy] allow_forwarded_ips setting in config.ini for more details.
      *
-     * @var bool
+     * @var string|bool
      */
     protected $allowForwardedIps;
 
     /**
      * Constructor
      *
      * @param Parameters $server            Server parameters
-     * @param bool       $allowForwardedIps Should we respect the X-Forwarded-For
-     * header?
+     * @param bool       $allowForwardedIps Forwarded header configuration string
```

Comment:
This probably should be @param string|bool.

Reply:
Thanks for the catch; fixed here: https://github.com/vufind-org/vufind/commit/9202ef95dd28be398630d262c79219cf8ef8b0a3